<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoeEditor</class>
 <widget class="QDialog" name="ShoeEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shoe Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="shoeName"/>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Brand</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="shoeBrand"/>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>121</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QLineEdit" name="shoeModel"/>
      </item>
      <item row="2" column="5">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QLineEdit" name="shoeSize"/>
      </item>
      <item row="3" column="5">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Notes</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLineEdit" name="shoeNotes"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Start  use</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>189</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="3" colspan="3">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>195</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QGroupBox" name="activeBox">
        <property name="title">
         <string>Active</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>End use</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="endUse">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QDateEdit" name="startUse">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Start mileage</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Max mileage</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="RunLineEdit" name="startMileage"/>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="RunLineEdit" name="maxMileage"/>
      </item>
      <item row="5" column="5">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="5">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RunLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/gui/runlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShoeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShoeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
